README.md

```diff
@@ -10,6 +10,7 @@ The Play Ebean plugin supports several different versions of Play and Ebean.
 
 | Plugin version | Play version | Ebean version |
 |----------------|--------------|---------------|
+| 4.1.1          | 2.6.x        | 11.15.x       |
```

Comment:
@KoenDG add this new line instead of changing the existing one because this table is listing releases, not branches.

Reply:
Ah, my bad there.